colaraz/ecommerce/colaraz-ecommerce/templates/edx/partials/_base_navbar.html

```diff
@@ -0,0 +1,49 @@
+{% load i18n %}
+{% load static %}
+
+<header class="navbar navbar-default">
+  <div class="container">
+    <!-- Brand and toggle get grouped for better mobile display -->
```

Comment:
@danialmalik This file is compatible with this commenting convention, as you can see that it is using this convention. I think it's jinja syntax not mako.